pkg/controller/combo_controller.go

```diff
@@ -153,6 +155,41 @@ func (c *combinationController) Reconcile(ctx context.Context, req ctrl.Request)
 		Message:            "evaluations successfully processed",
 	})
 
+	// If manifest application is indicated, create generated manifests on-cluster.
+	if combination.Spec.Apply{
```

Comment:
Maybe it makes sense to add a `v1alpha.ReasonProcessing` status update call here before the loop starts in case it takes awhile.